### DISCUSSION TITLE
Failed to pull setup Docker image

### DISCUSSION THREAD

**Discussion by alejandrocs98** [alshedivat/al-folio#2501] [Q&A] [answered]
Hi!
I followed all the [Installation and Deploying](https://github.com/george-gca/multi-language-al-folio/blob/main/INSTALL.md) instructions and everything worked fine in the remote repo. But I am getting the following error when trying to do the Local setup using Docker:
```{bash}
docker compose pull
```
```
unable to get image 'georgegca/multi-language-al-folio:latest': permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get "http://%2Fvar%2Frun%2Fdocker.sock/v1.45/images/georgegca/multi-language-al-folio:latest/json": dial unix /var/run/docker.sock: connect: permission denied
```

**Comment by george-gca** [comment, marked as answer]
This is an issue with your docker installation. See [this answer on stackoverflow](https://stackoverflow.com/a/48957722/14459430).
